src/analysis/calculus/times_cont_diff.lean

```diff
@@ -1943,59 +1957,143 @@ begin
   exact times_cont_diff_on_fderiv_within_apply hf unique_diff_on_univ hmn
 end
 
-/-- The sum of two `C^n`functions on a domain is `C^n`. -/
-lemma times_cont_diff_on.add {n : with_top ℕ} {s : set E} {f g : E → F}
-  (hf : times_cont_diff_on 𝕜 n f s) (hg : times_cont_diff_on 𝕜 n g s) :
-  times_cont_diff_on 𝕜 n (λx, f x + g x) s :=
+/-! ### Sum of two functions -/
+
+/-- The sum of two `C^n`functions within a set at a point is `C^n` within this set
```

Comment:
```suggestion
/-- The sum of two `C^n` functions within a set at a point is `C^n` within this set
```
Also fix this in the doc strings below.

Reply:
Good catch, thanks! Fixed (as well as two other occurences of such a missing space in mathlib)